test/tests/functional/pbs_equiv_classes.py

```diff
@@ -1536,11 +1536,14 @@ def test_preemption(self):
 
         a = {'Resource_List.ncpus': 3, 'queue': 'expressq'}
         (jid3,) = self.submit_jobs(1, a)
-
-        self.server.expect(JOB, {'job_state': 'R'}, id=jid1)
-        self.server.expect(JOB, {'job_state': 'S'}, id=jid2)
         self.server.expect(JOB, {'job_state': 'R'}, id=jid3)
 
+        # Make sure one of the job is suspended
+        sus_job = self.server.select(attrib={'job_state': 'S'})
+        if len(sus_job) != 1:
+            self.logger.info("Either no or more jobs are suspended")
+            self.assertTrue(False)
```

Comment:
Add the message to the assert, so it shows up in the list of errors to why it failed.